libvirt/resource_libvirt_domain_test.go

```diff
@@ -58,9 +58,10 @@ func TestAccLibvirtDomain_Detailed(t *testing.T) {
 			{
 				Config: fmt.Sprintf(`
 				resource "libvirt_domain" "%s" {
-					name   = "%s"
-					memory = 384
-					vcpu   = 2
+					name        = "%s"
```

Comment:
I would add here a seperate testcase for granularity and separation of concern

Reply:
ok, i will split this test in a few days :)

Reply:
thx!

Reply:
just ping me when done

Reply:
I extracted Description check into separate test.
The travis failed, but I think that my check has passed and it failed later on some storage test - maybe some random fail?

Reply:
let me check